src/main.rs

```diff
@@ -337,13 +341,14 @@ async fn check_in(id: String, state: &State<St>) -> Json<CheckInResp> {//Json<Ch
                     }
                 };
 
-                (person_name, gnum)
+                (person_name, gnum, Some(String::from("dfs")))
             };
 
             // TODO: try to insert member into members table
             let _ = diesel::insert_into(schema::members::table).values(Member {
                 gnum: gnum as i32,
-                is_staff: false
+                is_staff: false,
+                music: None,
```

Comment:
Right now there's no way for a user to change this field. We can fix this in a follow-up PR that adds upload functionality